internal/pkg/keeper/api/caller.go

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright (C) 2024 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package api
+
+import (
+	"errors"
+
+	"github.com/edgexfoundry/go-mod-configuration/v3/internal/pkg/keeper/utils/http"
+)
+
+type Caller struct {
+	baseUrl string
+}
+
+// NewCaller creates an instance of Caller
+func NewCaller(baseUrl string) *Caller {
+	return &Caller{
+		baseUrl: baseUrl,
+	}
+}
```

Comment:
```suggestion
// NewKeeperApi creates an instance of KeeperApi
func NewKeeperApi (baseUrl string) *KeeperApi{
	return &KeeperApi{
		baseUrl: baseUrl,
	}
}
```

Reply:
File removed, using the http clients from core-contract.